cpp/dolfin/io/cells.h

```diff
@@ -46,24 +46,26 @@ std::vector<std::uint8_t> lex_to_tp(mesh::CellType type, int num_nodes);
 /// @return The map
 std::vector<std::uint8_t> vtk_to_dolfin(mesh::CellType type, int num_nodes);
 
-/// Convert gmsh cell ordering to FEniCS cell ordering
+/// Convert VTK cell ordering to FEniCS cell ordering
 /// @param[in] cells array containing cell connectivities in VTK format
 /// @param[in] type Celltype to the permuter
 /// @return Permuted cell connectivities
 Eigen::Array<std::int64_t, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>
-gmsh_to_dolfin_ordering(
+vtk_to_dolfin_ordering(
+    const Eigen::Ref<const Eigen::Array<
+        std::int64_t, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor>>& cells,
+    mesh::CellType type);
+
+/// Convert Lexicographic cell ordering to FEniCS cell ordering
```

Comment:
Can the lexicographic be defined, maybe with a diagram? In the past it has sometimes increased fastest in the x-directions, and sometimes in the y-direction.

Reply:
Actually, that shouldn't matter. *Local* orientations are equivalence classes, i.e. all local orientations which are equal up to rotations are reflections should be considered the same. Or in other words, local orientation is a topological property.